util/feature.txt

```diff
@@ -0,0 +1,5 @@
+feature1 implementation
+
+feature2 implementation
```

Comment:
Let's add a bit more to the feature2 implementation. I think it should handle multiple weights

Reply:
@Juan-Barrantes see the latest commit for the implementation